src/main/java/com/openshift/internal/restclient/okhttp/ResponseCodeInterceptor.java

```diff
@@ -56,7 +61,9 @@ public Response intercept(Chain chain) throws IOException {
 				response = makeSuccessIfAuthorized(response);
 				break;
 			default:
-				throw createOpenShiftException(client, response, null);
+				if ( response.request().tag() instanceof Ignore == false ) {
```

Comment:
For websocket calls, if the interceptor throws an exception, it is swallowed up in the OkHttp Dispatcher. It is never delivered to WebSocketListener.onFailure. In order for ExecOutputListenerAdapter to indicate failure reliably, I needed a way to have the interceptor ignore my requests.